packages/paste-website/src/components/component-overview-table/index.tsx

```diff
@@ -29,7 +29,7 @@ const ComponentOverviewTable: React.FC<ComponentOverviewTableProps> = ({category
     .filter(({node}) => node.status === PackageStatus.BACKLOG)
     .sort(sortNodeByName);
   const sortedComponentsList = componentsList
-    .filter(({node}) => node.status !== PackageStatus.BACKLOG)
+    .filter(({node}) => node.status !== PackageStatus.BACKLOG && node.name !== '@twilio-paste/typography')
```

Comment:
This line made me notice that the typography meta-package is in the components folder.  It should probably be a sibling to it, like the core package.  No? @SiTaggart thoughts appreciated here too

Reply:
Yeah, potentially actually.

Would core still include typography, heading, paragraph and list? Or (my thinking) should typography just be it's own package outside of core, and core is just a bundle of the individual components?

Reply:
Further reasoning is I noticed there's a bit of an exported name clash when you build and deploy core. 

```
core
  exports typography
    exports heading
    exports paragraph
  exports heading
  exports paragraph
```
so heading and paragraph clash (or something, forget the exact warning)

Reply:
I'm starting to feel like the typography package is unnecessary complexity.  `core` and individual packages should be sufficient.

Reply:
> I'm starting to feel like the typography package is unnecessary complexity.

Now we're talking, simplify. I lean towards removing too

Reply:
I agree on this one. May have one been a good idea, but unnecessary now with `core`. So we need a ticket to deprecate the `typography` package? Any others?

Reply:
I think just Typography.  We need to remove any references to it from other packages/docs too

Reply:
Jira ticket created. I'm going to close this PR and open another one to remove references from other packages/docs.